tests/ApiTests/TestCase.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace Level23\Dynadot\Tests\ApiTests;
+
+class TestCase extends \PHPUnit\Framework\TestCase
+{
+}
```

Comment:
no tests?

Reply:
This is just a simple base class. Other test classes extend this one, which allow us to set general methods/logic in this class.

Reply:
I would suggest making it abstract in that case

Reply:
I agree, I have changed it.